OutSystems/outsystems-datagrid-reactive
# RGRIDT-663: Adding more API methods

This PR is for RGRIDT-663


### What was happening
* There were no API methods to activate, deactivate or clear filter. And there was no method to clear sort



### What was done
* Created 3 API methods for FIlter: Activate, Deactivate and Clear. These methods expect gridID and column ID as parameters.

* Created 1 API method for Sort: Clear. This method expects gridID as parameter.
* Created public client action for all these methods.

### Test Steps
1. Open Sample
2. Press Deactivate Filter Product Name button

Expected: Filter icon should disappear on Product Name column

1. Open Sample

2. Press Activate Filter Product Name button

Expected: Filter icon should appear on Product Name column

1. Open Sample

2. Filter for any value on Product Name column

3. Press Clear Filter Product Name button

Expected: Filter should be cleared and all values of the column should be shown.


1. Open Sample

2. (Multi) sort columns on grid.

3. Press Clear Sort button

Expected: Sort on all columns should be cleared and values should be displayed on original order.

### Checklist
